Sources/Models/Views/Controls/Slider.swift

```diff
@@ -51,6 +52,7 @@ public struct Slider: IBDecodable, ViewProtocol, IBIdentifiable {
                 switch key {
                 case .isMisplaced: return "misplaced"
                 case .isAmbiguous: return "ambiguous"
+                case .stateProperty: return "slider_value"
```

Comment:
If there is slider_value let's call it `sliderValue`, so here the line must be removed